### ISSUE TITLE
Display 10 as T

### ISSUE THREAD

**Issue by ADavidBailey** [EdwardPiwowar/BBA#956]
This is not Important.  But, in my opinion, an improvement.

**Comment by EdwardPiwowar** [comment]
Explain or any sample.

**Comment by ThorvaldAagaard** [comment]
![image](https://github.com/user-attachments/assets/0ef78119-b474-4783-892a-4f14bcafe1e7)

Replace 10 with T

**Comment by EdwardPiwowar** [comment]
T is the internal symbol for the card, which is 10 to humans. BBO and others use 10.

**Comment by ThorvaldAagaard** [comment]
I agree if you show it as cards, but in above diagram you do now show it as cards (ie each card should take the same space) like
![image](https://github.com/user-attachments/assets/c8e4a924-ed7a-453b-b3c9-26937de0cb1e).

But as David said: It is not important

Personally i prefer this

![image](https://github.com/user-attachments/assets/c39f3bc0-8b6a-4838-97a7-5fb64883461a)

**Comment by ADavidBailey** [comment]
> T is the internal symbol

It's the internal symbol for a reason -- it takes one cell.  Humans have to translate the two symbols to ONE card.  We're pretty efficient with that; but, it's easier as a single symbol.

**Comment by ADavidBailey** [comment]
I'm reminded of the Oil Company Board of Directors story.  The Board debated where to put the bicycle rack for over an hour and approved the new refinery in 10 minutes.

**Comment by EdwardPiwowar** [comment]
Pavlicek's view:
"“T” Versus “10”
A common practice, especially in the computer age, is to use the letter T to represent 10. The advantage is to allow each card to be represented by a single character, which simplifies some programming tasks and layout problems. Nonetheless, this policy will alienate many readers. The actual playing card has the number 10, not the letter T, so the association is clouded. (Note that higher playing cards are indeed marked by a letter, so the use of A-K-Q-J is clear.)

My feeling is that the letter T, while fine for database purposes, should be avoided in bridge writing. The only exception would be for informal writing when cards are given without spacing."

http://www.rpbridge.net/7z69.htm

**Comment by ThorvaldAagaard** [comment]
Reading some bridge articles I have no doubt, that I prefer T, both for reading and for programming. Even on the playing cards :-)

It is like the discussion about presenting a card as QS or SQ. Queen of Spades or Spade Queen. In Denmark we use the spoken term of Spade Queen so I have no doubt, but actually I prefer S4 instead of 4S, as 4S can be both a bid and a card, where S4 is a card.
